telemetry/infoFieldsMap.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Mappings of new unified-ping fields to classic-ping "info" fields
+
+# Maps unified-ping.application.* => classic-ping.info.*
```

Comment:
The arrow was documenting the operations to be done with the map, not the map itself. I agree your intepretation is likely to be more common